subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -517,7 +518,25 @@ int nrf_cloud_rest_location_get(struct nrf_cloud_rest_context *const rest_ctx,
 	memset(&resp, 0, sizeof(resp));
 	init_rest_client_request(rest_ctx, &req, HTTP_POST);
 
-	req.url = request->disable_response ? API_GET_LOCATION_NO_REPLY : API_GET_LOCATION;
+	char url[sizeof(API_GET_LOCATION_CONFIG)];
+	bool do_reply;
+
+	if (request->config) {
+		ret = snprintk(url, sizeof(url) - 1, API_GET_LOCATION_CONFIG,
+			       request->config->do_reply,
+			       request->config->fallback,
+			       request->config->hi_conf);
```

Comment:
Hmm, this client is in HTTP_POST mode, but we are using GET-style URL parameters? It's not **_impossible_** to do that, but it does seem odd

Reply:
This is the REST API doc for it:
https://api.nrfcloud.com/v1/#tag/Ground-Fix/operation/GetLocationFromCellsOrWifiNetworks